mgmtfn/k8splugin/README.md

```diff
@@ -0,0 +1,311 @@
+# Contiv Networking for Kubernetes
+
+Contiv is integrated with Kubernetes via a CNI plugin. This integration enables you
+to set up and manage inter-pod connectivity in a Kubernetes cluster using Contiv 
+policy model.
+
+## Getting Started
+
+This step-by-step procedure will guide you through a minimal experience of creating
+a Kubernetes cluster with Contiv networking and applying policy between pods.
+
+#### Step 1: Clone and build kubernetes
+
+```
+$ mkdir ~/go/src/github.com/k8s
+$ cd ~/go/src/github.com/k8s
+$ git clone https://github.com/kubernetes/kubernetes
+$ cd kubernetes; make
+```
+
+Note: If the build fails, please retry. It will usually build the binaries on 2nd attempt
```

Comment:
Suggest removing 'It will usually build the binaries on 2nd attempt". If at all something like this is needed "Often intermittent connectivity issues can result into a build failure, which is easily fixed by retrying the build"